playbooks/common/openshift-cluster/upgrades/v3_1_to_v3_2/pre.yml

```diff
@@ -100,8 +100,8 @@
   hosts: oo_masters_to_config:oo_nodes_to_config
   vars:
     target_version: "{{ '1.2' if deployment_type == 'origin' else '3.1.1.900' }}"
-    openshift_docker_hosted_registry_insecure: True
-    openshift_docker_hosted_registry_network: "{{ hostvars[groups.oo_first_master.0].openshift.common.portal_net }}"
+    docker_hosted_registry_insecure: "{{ openshift_docker_hosted_registry_insecure | default(True) }}"
```

Comment:
> I believe we need to make sure that this setting doesn't get exposed directly in the docker role, since it is an OpenShift specific configuration item, the hosted registry belongs to OpenShift (openshift_docker) and not Docker.

By "exposed directly in the docker role" do you mean "used in" the docker role? (slightly confused on the terminology) My understanding, playbook includes a role, we pass it settings, those go into a semi-global namespace accessible to all dependent roles included so it would therefore be exposed to the docker role. Wondering if that's right and if so how we avoid that.